snorkel/labeling/model/label_model.py

```diff
@@ -892,6 +892,9 @@ def fit(
         # Train the model
         metrics_hist = {}  # The most recently seen value for all metrics
         for epoch in range(start_iteration, self.train_config.n_epochs):
+            if self.config.verbose:
+                print("Epoch {0}/{1}".format(epoch+1, self.train_config.n_epochs))
```

Comment:
Can we use `logging` statements here?

We have a logging utility here, as an example: https://github.com/snorkel-team/snorkel/blob/master/snorkel/labeling/model/label_model.py#L602